components/data-feed-service/service/data-feed-aggregate-task_test.go

```diff
@@ -0,0 +1,41 @@
+package service
+
+import (
+	"context"
+	"testing"
+)
+
+const (
+	ipAddress = "192.168.10.10"
+	nodeId    = "e5a3d952-5dfc-3215-bfe4-541eb0b5d345"
+)
+
+func TestIsIpaddressTrue(t *testing.T) {
+	isIPAddress := isIPAddress(ipAddress)
+	if !isIPAddress {
+		t.Log("Expected true got false")
+		t.Fail()
+	}
```

Comment:
If we wanted to use https://github.com/stretchr/testify here, we could make this
```suggestion
	assert.True(isIPAddress(ipAddress))
```
If we want to avoid the dependency (but really, it's used all over Automate already), we could do
```suggestion
	if !isIPAddress(ipAddress) {
		t.Error("expected true got false")
	}
```
[since](https://golang.org/pkg/testing/#T.Error) _"Error is equivalent to Log followed by Fail"_